src/modules/txbuilder/RadixTransactionBuilder.ts

```diff
@@ -776,6 +776,8 @@ export default class RadixTransactionBuilder {
             .then(connection => {
                 RadixTransactionBuilder.signAndSubmitAtom(atom, connection, signer)
                     .subscribe(stateSubject)
+            }).catch(e => {
+                logger.error(e)
```

Comment:
I think this needs to do more than just catch and log it - we can easily propagate the error to the user through the stateSubject